packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -52,15 +53,9 @@ function Spec(attrs) {
     function() {
       return '';
     };
-  this.expectationResultFactory = attrs.expectationResultFactory ||
-    function() {};
   this.queueRunnerFactory = attrs.queueRunnerFactory || function() {};
   this.throwOnExpectationFailure = !!attrs.throwOnExpectationFailure;
 
-  if (!this.queueableFn.fn) {
-    this.pend();
```

Comment:
yeah, we have this logic already
https://github.com/facebook/jest/blob/master/packages/jest-jasmine2/src/jasmine-async.js#L59